docs/lib/project-setup/fragments/flutter/create-application/40_verifyAmplifyLibraries.md

```diff
@@ -46,3 +46,9 @@ class _MyHomePageState extends State<MyHomePage> {
 ```
 
 Note that all calls to `addPlugin()` or `addPlugins()` are made before `Amplify.configure()` is called.
+
+If you need to test whether Amplify has already been configured successfully, you can use the `isConfigured` getter:
+
+```dart
+ var isConfigured = Amplify.isConfigured;
```

Comment:
general question, I was playing with Amplify/Flutter and realized that in our docs we're not very consistent when declaring variables, like recommended here: https://dart.dev/guides/language/effective-dart/usage#do-follow-a-consistent-rule-for-var-and-final-on-local-variables

it seems like `final` with type inference should be used more often, but I see `var` and the explicit type declared often in our snippets. Do we follow a pattern?

Reply:
@drochetti We probably do not follow a consistent pattern (but obviously we should).